src/mysql-util/src/tunnel.rs

```diff
@@ -270,6 +274,18 @@ impl Config {
     ) -> Result<MySqlConn, MySqlError> {
         let mut opts_builder = OptsBuilder::from_opts(self.inner.clone());
 
+        if let Some(aws_config) = &self.aws_config {
+            let (host, port) = self.address();
+            let username = self.inner.user().expect("MySQL: username required");
+
+            let token = rds_auth_token(host, port, username, aws_config).await?;
+            // Cleartext plugin must be enabled for IAM authentication, for security,
```

Comment:
Does the user need to do anything to support the "cleartext" plugin? If so would be great to call it out in some docs!